test/test_torch.py

```diff
@@ -17373,6 +17375,67 @@ def test_hypot(self, device, dtype):
                 expected = np.hypot(input[0].cpu().numpy(), input[1].cpu().numpy())
             self.assertEqual(actual, expected)
 
+    def _helper_test_igamma(self, loglo, loghi, device, dtype):
+        exp1 = 2.71828182846
+        vec1 = torch.logspace(loglo, loghi, steps=500, base=exp1,
+                              dtype=torch.float64, device=device).unsqueeze(-1)
+        vec1 = vec1.to(dtype)
+        inputs = [
+            (vec1, vec1.transpose(0, 1)),
+            (vec1, vec1),  # for large number, it should approach 0.5
+            (vec1, 0.5 * vec1),  # test for considerable ratio
+            (vec1, 2.0 * vec1),
```

Comment:
Add two test cases where each input is a scalar:

```
(.378, vec1),
(vec, .018)
```

Or similar.

Reply:
It's not working because `TypeError: igamma(): argument 'input' (position 1) must be Tensor, not float`.
I'm not sure how to make torch function to receive bare scalar (without converting it manually to Tensor first).
Or did you mean scalar tensor?

Reply:
Oh right, sorry. Some of our functions accept scalars and I should have been clearer. For many of them we have to add explicit scalar overloads.

What about on CUDA, does putting a scalar in work there?

Reply:
No, it doesn't work either.

Reply:
OK. Sounds good. Please ignore this suggestion, then.